pkg/utils/validation/cidr/disjoint.go

```diff
@@ -33,6 +34,9 @@ func ValidateNetworkDisjointedness(fldPath *field.Path, shootNodes, shootPods, s
 	if shootNodes != nil && seedNodes != nil && NetworksIntersect(*shootNodes, *seedNodes) {
 		allErrs = append(allErrs, field.Invalid(pathNodes, *shootNodes, "shoot node network intersects with seed node network"))
 	}
+	if shootNodes != nil && seedNodes != nil && NetworksIntersect(*shootNodes, constants.DefaultVpnRange) {
+		allErrs = append(allErrs, field.Invalid(pathNodes, *shootNodes, "shoot node network intersects with default vpn network ("+constants.DefaultVpnRange+")"))
```

Comment:
Use `fmt.Sprintf`?

Reply:
Sure, I can change it accordingly. Why is there a preference for sprintf compared to "ordinary" string concatenation?

Reply:
readability

Reply:
Done.